meltano.yml

```diff
@@ -1,26 +1,43 @@
 version: 1
-send_anonymous_usage_stats: true
+send_anonymous_usage_stats: false
 project_id: 96584f7b-a36c-46e0-b41a-7f9074293137
 plugins:
   extractors:
   - name: tap-github
     namespace: tap_github
-    pip_url: blah
-    executable: /Users/ajsteers/Source/tap-github/tap-github.sh
+    pip_url: -e .
     capabilities:
     - state
     - catalog
     - discover
     settings:
-    - name: searches
-      kind: array
+    - name: user_agent
+      kind: string
+    - name: metrics_log_level
+      kind: string
     - name: auth_token
       kind: password
+    - name: additional_auth_tokens
+      kind: array
+    - name: rate_limit_buffer
+      kind: array
```

Comment:
should this also be updated @edgarrmondragon ?

Reply:
it should 😄